docs/next/util/limit.ts

```diff
@@ -50,8 +50,9 @@ export const limitSnippetLines = (content, fromTo, dedent, startAfter, endBefore
   }
 
   const dedentedElements = elements.map((x) => x.substring(dedentLevel));
+  const filteredElements = dedentedElements.filter((x) => !x.trim().startsWith('# type: ignore'));
```

Comment:
The annotations go at the end of the line, not on their own line. I think we need to take the substring up until we see `# type: ignore`

Reply:
you're totally right - fixed